tests/DependencyInjection/RabbitMqConsumerHandlerExtensionTest.php

```diff
@@ -80,29 +80,56 @@ public function testDefaultConfigurationServices(string $aliasName, string $targ
 		$this->compile();
 	}
 
-	public function testConfigureStopConsumerSleepSeconds(): void
+	/**
+	 * @return mixed[][]|\Generator
+	 */
+	public function configureContainerParameterDataProvider(): Generator
```

Comment:
With this I think we can incorporate the cases from `defaultConfigurationValuesDataProvider()` here - maybe just directly, so we do not have to somehow compose the case names.

Reply:
ok, good idea :), incorporated

Reply:
Now it is pretty duplicated, that's why I wrote directly :)

Reply:
Ok, I see, it should be deduplicated properly now :)